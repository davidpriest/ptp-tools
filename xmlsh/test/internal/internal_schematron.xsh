# Test schematron validation
# same as xvalidate -schematron

echo Schematron validation
N=$(schematron ../../samples/data/books.sch ../../samples/data/books.xml | xmlns:svrl=http://purl.oclc.org/dsdl/svrl xpath 'count(//svrl:fired-rule)')
echo Matching Items: $N
