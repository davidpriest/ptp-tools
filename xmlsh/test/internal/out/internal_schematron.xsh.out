Schematron validation
Matching Items: 6
